### PR TITLE
Update to support carthage

### PR DESCRIPTION
**Pull request by pisces**
New files to support carthage
Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +76,25 @@ it, simply add the following line to your Podfile:
...
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
+
+You can install Carthage with [Homebrew](http://brew.sh/) using the following command:
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
```
Comment (akramhussein) [inline README.md]:
```suggestion
To integrate Mute into your Xcode project using Carthage, specify it in your `Cartfile`:
```

**Comment by akramhussein** [comment]
Hi @pisces - thanks for this PR. I've not used `Carthage` before (but know of it). I'll take a deeper look and test it out. 

Are these project file changes also necessary to get it working?

File: README.md (hunk 1 above)
Comment (pisces) [reply README.md]: @akramhussein 
that is mistake caused by copy paste, I will fix it.

**Comment by pisces** [comment]
@akramhussein 
What the project file you said?

**Comment by akramhussein** [comment]
> @akramhussein
> What the project file you said?

`Mute.xcodeproj/project.pbxproj`

**Comment by pisces** [comment]
@sk

> > @akramhussein
> > What the project file you said?
> 
> `Mute.xcodeproj/project.pbxproj`

Yes, it is needed, to build dynamic framework for carthage:)

**Comment by akramhussein** [comment]
No problem - will try and test it all tonight.

**Comment by pisces** [comment]
> No problem - will try and test it all tonight.

Thanks for your feedback.

**Comment by pisces** [comment]
@akramhussein 
When does the test finish?

**Comment by akramhussein** [comment]
Hi - I got around to testing this PR. 

It does seem to work, but I had issues running in Simulator and with Xcode 1).

It seems related to this [issue](https://stackoverflow.com/questions/64026065/xcode-12-issue-could-not-find-module-frameworkname-for-target-arm64-apple-i). Would it be possible to take a look to see if this can be resolved?

**Comment by pisces** [comment]
@akramhussein
Yes. I have been resolve that issue in the past, will push new commit for that soon:)

**Comment by pisces** [comment]
@akramhussein 
A little late.
I did add new example project for carthage and write shell script file.

**Comment by akramhussein** [comment]
Hi @pisces - I will take a look at this. Thanks for working on it. At first glance though there are some references to your account e.g. `https://github.com/pisces/Mute.git` which isn't ideal since it may not run for other people.

**Comment by pisces** [comment]
Hi @akramhussein 
Referece is changed to `https://github.com/akramhussein/Mute.git` in Cartfile, but carthage cannot install in ExampleForCarthage  before merge.

**Pull request closed without merging**